src/olympia/lib/settings_base.py

```diff
@@ -85,7 +85,7 @@ def path(*folders):
 FLIGTAR = 'amo-admins+fligtar-rip@mozilla.org'
 THEMES_EMAIL = 'theme-reviews@mozilla.org'
 ABUSE_EMAIL = 'amo-admins+ivebeenabused@mozilla.org'
-NOBODY_EMAIL = 'nobody@mozilla.org'
+ADDONS_EMAIL = 'nobody@mozilla.org'
```

Comment:
`ADDONS_EMAIL` already exists elsewhere in the settings file, so you can just delete this line.

Reply:
Done :)